src/autoplex/auto/phonons/jobs.py

```diff
@@ -619,3 +796,60 @@ def get_iso_atom(
         },
     )
     return Response(replace=flow)
+
+
+@job(data=[PhononBSDOSDoc, "dft_references"])
+def get_output(
+    metrics: list,
+    benchmark_structures: list[Structure] | None = None,
+    benchmark_mp_ids: list[str] | None = None,
+    dft_references: list[PhononBSDOSDoc] | None = None,
+    pre_xyz_files: list[str] | None = None,
+    pre_database_dir: str | None = None,
+    fit_kwargs_list: list | None = None,
+):
+    """
+    Job to collect all output infos for potential restarts.
```

Comment:
the only thing I see code-wise is that this function name and its description coule be more descriptive. I needed to look at the unit test to fully get it 😃

Reply:
Feel free to add more description 😀